mlflow/sklearn/utils.py

```diff
@@ -771,7 +771,9 @@ def _backported_all_estimators(type_filter=None):
     import sklearn
     from importlib import import_module
     from operator import itemgetter
-    from sklearn.utils.testing import ignore_warnings  # pylint: disable=no-name-in-module
+
+    # pylint: disable=no-name-in-module, import-error
+    from sklearn.utils.testing import ignore_warnings
```

Comment:
@harupy Is it sufficient to fix the linting here or do we need a replacement for `ignore_warnings` against scikit-learn 0.24.0? Naively, it seems like this will still throw an import error and hamper the usage of scikit-learn 0.24.0, yet our tests pass....

Reply:
After online discussion from @harupy, this is acceptable since `_backported_all_estimators` is only used for older versions of scikit-learn.